pycascadia/remove_restore.py

```diff
@@ -13,13 +12,17 @@
 
 from pycascadia.loaders import load_source
 from pycascadia.grid import Grid
-from pycascadia.utility import region_to_str, min_regions
+from pycascadia.utility import region_to_str, min_regions, is_region_valid
 
 def calc_diff_grid(base_grid, update_grid, diff_threshold=0.0, window_width=None):
     """Calculates difference grid for use in remove-restore"""
     print("Blockmedian update grid")
     max_spacing = max(update_grid.spacing, base_grid.spacing)
     minimal_region = min_regions(update_grid.region, base_grid.region)
+    if not is_region_valid(minimal_region):
+        print("Update grid is outside region of interest. Skipping.")
```

Comment:
If I understand correctly, we could make this clearer by
```suggestion
        print("Update grid is (at least partially) outside region of interest. Skipping.")
```

Reply:
If it's only partially outside the ROI its data will be included. This check is only for when the update grid is *entirely* outside the ROI and can be skipped.

Reply:
Right, yes, OK, agreed 🤦 
Then maybe? 😆 
```suggestion
        print("Update grid is entirely outside region of interest. Skipping.")
```